### ISSUE TITLE
@tabrindle it looks like 7.7.2 thinks it's 7.7.1 ?

### ISSUE THREAD

**Issue by asklar** [tabrindle/envinfo#183]
@tabrindle it looks like 7.7.2 thinks it's 7.7.1 ?
![image](https://user-images.githubusercontent.com/22989529/88249577-b0e25500-cc59-11ea-85f6-6afb50443b9b.png)

_Originally posted by @asklar in https://github.com/tabrindle/envinfo/pull/180#issuecomment-662801350_

**Comment by asklar** [comment]
![image](https://user-images.githubusercontent.com/22989529/88314642-12420c80-ccca-11ea-980d-a2f6332f6da2.png)

**Comment by tabrindle** [comment]
yeah this is a quirk with the webpack process, definitely needs addressed. A bit ironic, isn't it? :-|
